src/components/common/MuiMarkdown.tsx

```diff
@@ -56,7 +65,7 @@ class MuiMarkdown extends React.PureComponent {
           },
         }}
       >
-        {this.props.children}
+        {this.sanitisedMarkdown}
```

Comment:
This doesn't work as it doesn't change the used markdown when language is change, you have to make it part of the state.

Reply:
Fixed with https://github.com/gesund-zusammen/gesund-zusammen.github.io/pull/93/commits/19cbf62a6b8faec2feeab2f33e1fabe386a74477